cura/CuraAppSignals.py

```diff
@@ -0,0 +1,9 @@
+from PyQt5.QtCore import pyqtSignal, QObject
+
+
+class CuraAppSignals(QObject):
+
+    showMoreInfoOnAnonymousDataCollection = pyqtSignal()
```

Comment:
Please don't communicate this via a signal. That introduces unnecessary overhead and makes the code more complex than needed. A simple `pyqtSlot` should do the job. That also prevents the need to create code that's not in the SliceInfoPlugin.
For reference on how to use a pyqtSlot to create a dialogue you could look at the UraniumExampleExtensionPlugin or the Change Log.

Reply:
Fixed in 35e7be1d4. The problem is that this is in the Preferences page as well, so there needs to be a way to show the dialog from the Preferences. I think `SliceInfo` can later be changed to a built-in module instead of a plugin because of those dependecnies.